ci/customization/customize_docs_in_folder.sh

```diff
@@ -51,5 +51,5 @@ grep "${JTD_SEARCH_TERM}\|${DOXYGEN_SEARCH_TERM}\|${PYDATA_SEARCH_TERM}" -rl \
 > "${MANIFEST_FILE}"
 
 echo "Customizing $(wc -l < ${MANIFEST_FILE} | tr -d ' ') HTML files"
-python ${SCRIPT_SRC_FOLDER}/customize_doc.py "${MANIFEST_FILE}"
+python -u ${SCRIPT_SRC_FOLDER}/customize_doc.py "${MANIFEST_FILE}"
```

Comment:
For those finding this PR from search... adding `-u` ensures that Python stdout / stderr are printed immediately ("unbuffered").

I found that the warning about unrecognized sphinx themes, which goes to stderr, was not being printed immediately after the log messages about that same file printed to stdout.

Removing buffering fixes that.